requirements.txt

```diff
@@ -11,3 +11,5 @@ psycopg2-binary==2.7.5
 scikit-learn==0.19.2
 tqdm==4.11.2
 scikit-optimize==0.5.2
+Flask==1.0.2
```

Comment:
We should put it to a separate requirements.txt under `lookout/style/format/visualizer`